wiki0/src/server/main.ts

```diff
@@ -23,10 +23,16 @@ app.use(express.json());
 // TODO uncomment when ready
 // app.use('/.well-known', wellknown);
 
-const redisClient = createClient({
-  url: process.env.REDIS_SERVER,
-});
-redisClient.connect().catch(console.error);
+const redisClient = createClient();
+
+redisClient.on('error', (error) => console.error(`Redis Error : ${error}`));
+
+(async () => {
```

Comment:
actually at this point, just revert to the code that was there before. It's more concise:

`redisClient.connect().catch(console.error);`